exonum-java-binding/core/rust/exonum-java/README.md

```diff
@@ -0,0 +1,16 @@
+# Exonum Java Binding Application
+
+Exonum Java Binding is a framework for building blockchain applications in Java, 
+powered by Exonum.
+
+With Exonum Java Binding you can define stateful Exonum Services in Java,
+configure a network of nodes and run your services in it.
+
+For more information see the relevant documentation on the tool:
+  - [User Guide](https://exonum.com/doc/version/0.13/get-started/java-binding/)
```

Comment:
0.13-rc.2, see the https://wiki.bf.local/display/EJB/Java+Binding+0.9.0-rc1+Release+Checklist